system-tests grpc-js fail

This library does not use gax so should not have system tests grpc-js at all, but if we add that task here to make the setup similar, its configuration must be fixed. Right now the task is failing immediately with

```
[16:37:46] [ID: 5735578] Build finished after 1 secs, exit value: 0
[16:37:46] 
[16:37:46] 
[16:37:46] Command done in 10 seconds
[16:37:49][ERROR] Failed to get build config
com.google.devtools.kokoro.config.ConfigException: Couldn't find build configuration under /tmp/workspace/workspace/cloud-devrel/client-libraries/nodejs/continuous/googleapis/cloud-profiler-nodejs/node8/system-test-grpcjs/src/github/cloud-profiler-nodejs/.kokoro/continuous/node8.
```

@kinwa91 can you please take a look and fix? Thanks!

--- a/.kokoro/continuous/node8/system-test-grpcjs.cfg
+++ b/.kokoro/continuous/node8/system-test-grpcjs.cfg
@@ -1,0 +1,12 @@
+# Download resources for system tests (service account key, etc.)
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-nodejs"
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/cloud-profiler-nodejs/.kokoro/system-test.sh"
+}
+
+env_vars: {
+    key: "GOOGLE_CLOUD_USE_GRPC_JS"
+    value: "1"
+}

